src/content/developers/docs/consensus-mechanisms/poa/index.md

```diff
@@ -0,0 +1,69 @@
+
+## how it works {#how-it-works}
+
+clique standard simply make a use of the obsolete headers: extradata that was used only for fun, miner became obsolete as there are no miners in POA, and nonce became also obsolete because the authorities the order of users transactions, extradata is used to store the authorized signers set, miner and nonce are used to vote for adding or removing signers from the network, where every epoch transition flushes all pending votes. The block proposal is selected sequentially for each block, but in fact any other authorized signer can propose block even if it's not his turn, but they have limit of signing only one block between K blocks when N are the authorized signers.
+
+## PROOF-OF-AUTHORITY AND SECURITY {#poa-security}
+
+### Attack vector: Malicious signer
+
+It may happen that a malicious user gets added to the list of signers, or that a signer key/machine is compromised. In such a scenario the protocol needs to be able to defend itself against reorganizations and spamming. The proposed solution is that given a list of N authorized signers, any signer may only mint 1 block out of every K. This ensures that damage is limited, and the remainder of the miners can vote out the malicious user.
+
+### Attack vector: Censoring signer
+
+Another interesting attack vector is if a signer (or group of signers) attempts to censor out blocks that vote on removing them from the authorization list. To work around this, we restrict the allowed minting frequency of signers to 1 out of N/2. This ensures that malicious signers need to control at least 51% of signing accounts, at which case it’s game over anyway.
+
+### Attack vector: Spamming signer
+
+A final small attack vector is that of malicious signers injecting new vote proposals inside every block they mint. Since nodes need to tally up all votes to create the actual list of authorized signers, they need to track all votes through time. Without placing a limit on the vote window, this could grow slowly, yet unbounded. The solution is to place a ~~moving~~ window of W blocks after which votes are considered stale. ~~A sane window might be 1-2 epochs.~~ We’ll call this an epoch.
+
+### Attack vector: Concurrent blocks
+
+If the number of authorized signers are N, and we allow each signer to mint 1 block out of K, then at any point in time N-K+1 miners are allowed to mint. To avoid these racing for blocks, every signer would add a small random “offset” to the time it releases a new block. This ensures that small forks are rare, but occasionally still happen (as on the main net). If a signer is caught abusing it’s authority and causing chaos, it can be voted out.
```

Comment:
I'm not sure I understand from reading this... Maybe we could use some example numbers here to clarify what is meant?

"at any point in time N-K+1 miners are..." Can you clarify what is meant here?

Also, a general note, I would consider removing the use of the word "miner" throughout this page unless specifically talking about proof-of-work, or a comparison to it... I would use "validator" instead, or be even more specific depending on the topic, such as "block proposer" or "attester" 

"the main net" -> "Mainnet"

"it's authority" -> "its authority"